integration_tests/e2e/overview/workAndInterestsTab.cy.ts

```diff
@@ -307,5 +409,21 @@ context('Prisoner Overview page - Work and Interests tab', () => {
       // Then
       Page.verifyOnPage(InPrisonWorkPage)
     })
+
+    it(`should link to auth-error page given user has read-only authority`, () => {
```

Comment:
Ooooh, I think this is wrong, but is actually a sign of a different problem which I think we should fix first ....

So, all these Change links should only be shown if you have the EDITOR role ... but at the moment we show them regardless 😬 

I think we need to part this PR for the time being, and create a new branch and PR to fix up all the Change and Add links on the Education & Training, and Work Interests pages.

Basically we need to change all instances of this type of thing:
```
              <a class="govuk-link app-summary-card__change-link__link govuk-!-display-none-print" href="{{ educationalQualificationsChangeLinkUrl }}" data-qa="educational-qualifications-change-link">
                Change<span class="govuk-visually-hidden"> qualifications by adding or removing qualifications</span>
              </a>
```
and wrap it in an if block, like this:
```
            {% if hasEditAuthority %}
              <a class="govuk-link app-summary-card__change-link__link govuk-!-display-none-print" href="{{ educationalQualificationsChangeLinkUrl }}" data-qa="educational-qualifications-change-link">
                Change<span class="govuk-visually-hidden"> qualifications by adding or removing qualifications</span>
              </a>
            {% endif %}
```
You don't need to add the `hasEditAuthority` to all the View classes, it will already be there. The if statement above should work 🤞